src/harness/harnessLanguageService.ts

```diff
@@ -336,6 +336,9 @@ module Harness.LanguageService {
         getOccurrencesAtPosition(fileName: string, position: number): ts.ReferenceEntry[] {
             return unwrapJSONCallResult(this.shim.getOccurrencesAtPosition(fileName, position));
         }
+        getDocumentHighlights(fileName: string, position: number, filesToSearch: string[]): ts.DocumentHighlights[] {

```

Comment:
I deleted the comment because we discussed it offline. To make you look less insane, I was asking why it wasn't called `getReferenceHighlights`.